setconfig/RADIANT.json

```diff
@@ -1,13 +1,19 @@
 {
     "name": "RADIANT",
+
+    "default_args": {
+        "args": {
+            "use_uart": true
+        }
+    },
+
     "tests": {
         "SystemPower": {},
         "uCComms": {},
         "RADIANTPower": {},
         "FPGAComms": {},
         "LAB4DTune": {},
         "LAB4DGlitch": {},
-        "FrontEndNoise": {},
```

Comment:
Why that?

Reply:
I missed that, was not on purpose. I will add it again, thanks!